poetry/utils/venv.py

```diff
@@ -243,6 +243,8 @@ def run(self, bin, *args, **kwargs):
         call = kwargs.pop("call", False)
 
         if shell:
+            if self._windows:
+                cmd[0] = '"{}"'.format(cmd[0])
```

Comment:
two things:
* we should escape path on all platforms (`"` should work fine - `'` while safer on UNIX-like systems may indeed not work on windows, but I didn't check)
* we should most likely escape all arguments this way when transform list of args to a string for shell to interpret

Reply:
Regarding escaping paths on Windows:
* PowerShell seems to want either an ampersand or a dot in front of the escaped path for the executable to be executed `[&|.] '<path to exe>'`. If there's no ampersand/dot, it just echoes the path back.
* PowerShell is fine with both single and double quotes.
* CMD only wants the escaped path.
* CMD only wants double quotes.
Do note that this is based on me just trying stuff for myself. The actual details on escaping executable paths for PowersShell might be a little different.